### PR TITLE
Update Model docs for API-4994 and API-4995 (API-5003)

### PR DESCRIPTION
**Pull request by lukebversal**
### Summary
Update the following documentation and swagger configs to reflect the following changes to the API:
- Add a "roles" to `/courses/{courseId}/users/{userId}` (see [API-4995](https://versal.atlassian.net/browse/API-4995)) to the User_Tracking_Input_Model
- Add a "member" role to Org_User_Input_Model

### Refs
[API-5003](https://versal.atlassian.net/browse/API-5003)
[API-4994](https://versal.atlassian.net/browse/API-4994)
[API-4995](https://versal.atlassian.net/browse/API-4995)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -655,6 +656,7 @@ Returns a paginated result set.
 | Name | Type | Description |
 | ---- | ---- | ----------- |
 | tracked | boolean | Whether the user is tracked for the course |
+| roles | [ string ] | Accepted values: "learner", "author", "contributor", "publisher" |
```
Comment (pacharya) [inline README.md:667]: So we have course roles and org roles. The table in a different part of this PR explains org roles well. Course roles in top line, however, are not explained well. Do we want a table for them? also do we want to explicitly label them as course vs org roles? I can give you text for the above four course roles.

**Review by pacharya — APPROVED** [review]
just a question - feel free to ignore

File: README.md (hunk 1 above)
Comment (sdkarlson) [reply README.md:667]: 👍 good idea

Comment (pacharya) [reply README.md:667]: "learner" - A learner a learner in the course
 "author" - An author an org member with author privileges
 "contributor" - A contributor  an out-of-org user with author privileges
"publisher" - A publisher can publish newer revisions of the course - all learners only see the latest revision.

Comment (sdkarlson) [reply README.md:667]: "author" and "contributor" are synonyms, org membership doesn't enter into it...?

Comment (lukebversal) [reply README.md:667]: For clarity, should I just merge this PR and add the roles table in a separate PR once the final table verbiage is set?

Comment (pacharya) [reply README.md:667]: oh its fine.. lets go with

"author" / "contributor" can edit the course
"learner" can learn the course
"publisher" .. above

Please make updates and 🚢

Comment (lukebversal) [reply README.md:667]: See latest commit for "Roles" updates. Let me know if anything needs to be adjusted further.

**Review by pacharya — APPROVED** [review]

**Comment by pacharya** [comment]
🚢  when happy

**Pull request merged**